custom_components/ble_monitor/const.py

```diff
@@ -1120,7 +1121,6 @@ class BLEMonitorBinarySensorEntityDescription(
     'iBeacon'                 : 'Apple',
     'AltBeacon'               : 'Radius Networks',
     'EClerk Eco'              : 'Relsib',
-    'Air Mentor Pro 2'        : 'Air Mentor',
```

Comment:
Why is Air Mentor Pro 2 delete? I think you should add Acconeer.

Reply:
Sorry about that. Rebase conflict fix gone bad